lib/modules/platform/bitbucket/index.ts

```diff
@@ -113,17 +114,30 @@ export async function initPlatform({
 }
 
 // Get all repositories that the user has access to
-export async function getRepos(): Promise<string[]> {
+export async function getRepos(config?: AutodiscoverConfig): Promise<string[]> {
```

Comment:
```suggestion
export async function getRepos(config: AutodiscoverConfig): Promise<string[]> {
```
I think it's not optional and will always be passed from worker, so needs test updates.